### ISSUE TITLE
A11y: Button announced as group

### ISSUE THREAD

**Issue by tjameswhite** [Esri/calcite-components#1398]
### Summary
Similar to https://github.com/Esri/calcite-components/issues/646

When using Jaws, buttons are announced as a group. User must then enter group to get button text.

### Actual Behavior
Screen reader announces "button group" 

### Expected Behavior
Screen reader should announce the button text; eg: "submit"

### Relevant Info 
Tested Jaws Chrome on Windows 10

**Comment by julio8a** [comment]
For me it shows the text of the button and that it's a button:
![image](https://user-images.githubusercontent.com/6144462/125894793-ff5fa2d9-dccf-4be0-8e17-35c758603d07.png)

Can you check this again?
